opensquirrel/ir.py

```diff
@@ -247,14 +210,25 @@ def is_identity(self) -> bool:
         # Angle and phase are already normalized.
         return abs(self.angle) < ATOL and abs(self.phase) < ATOL
 
-    def relabel(self, mapping: Mapping[int, int]) -> None:
-        """Relabel the qubits using the given mapping.
+    def get_default_bloch(self) -> BlochSphereRotation | None:
```

Comment:
Hey Roberto, great pull request! Well done!

It may be interesting to name this function `wrap_default_bloch` and return a `BlochSphereRotation` only. Thus, if the function does not find a default bloch, it returns itself.

This may be better in the future and also help clear some lines in the general_merger.

Reply:
Good idea, yes, I prefer that than calling the get function and then check against none.

But then let's implement this as a free function receiving a BSR, instead of a BSR method, something like `try_name_anonymous_bloch(bsr: BSR) -> BSR`. That receives a BSR and returns the same BSR if it doesn't find a close default BSR, or a close default BSR otherwise.

Because we're not actually wrapping anything,  we're trying to convert an anonymous bloch into a default bloch.

We can put this free function in the merger,  for the time being, and invoke it as `blah = try_name_anonymous_bloch(blah)`.